### PR TITLE
Milestone #1

### PR DESCRIPTION
**Pull request by Farida-Hamid**
>In this project, we built a basic website that allows users to add/remove books from a list. 
We achieved that by:
- Using JavaScript objects and arrays,
- Modifying the DOM, and
- Adding basic events dynamically.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  // update local storage
+  localStorage.setItem('aboutBook', JSON.stringify(books));
+
+  show(books.length - 1);
+});
+
+// add DOM elements
+// const form = document.querySelector('#form');
+// const titleInput = document.querySelector('#title');
+// const authorInput = document.querySelector('#author');
```
Comment (omar-muhamad) [inline script.js]: - [x] Please remove any unused (commented) code to make your code cleaner and more readable.

File: README.md [hunk 2]
```diff
@@ -1 +1,101 @@
...
+![gh actions failing checks](../assets/images/gh-actions-html-css-failing-checks.png)
+
+## Set-up linters in your local env
+
+**Note**: The `npm` package manager is going to create a `node_modules` directory to install all of your dependencies. You shouldn't commit that directory. To avoid that, you can create a [`.gitignore`](https://git-scm.com/docs/gitignore) file and add `node_modules` to it:
+
+```
+# .gitignore
+node_modules/
+```
```
Comment (omar-muhamad) [inline README.md]: - [x] Please Edit your readme file by removing what's inside it now and adding (project name, description, built with, get started, authors, and license) sections, You can use this [Microverse template](https://github.com/microverseinc/readme-template) and edit it.

File: script.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    books = JSON.parse(localStorage.getItem('aboutBook'));
+  }
+}
+retrieve();
+
+const main = document.querySelector('.books');
+
+function addBook(enteredTitle, enteredAuthor) {
+  books.push({ title: enteredTitle, author: enteredAuthor });
+}
```
Comment (omar-muhamad) [inline script.js:17]: - [x] Please make sure that when a user clicks the "Add" button:
- A new book is added to the collection.
- The new book is displayed on the page.

File: script.js [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+  books = books.filter((element) => element !== books[item.getAttribute('data-id')]);
+  // Update books
+  localStorage.setItem('aboutBook', JSON.stringify(books));
+  // Initialize screen
+  main.innerHTML = '';
+  // Display books
+  for (let i = 0; i < books.length; i += 1) {
+    show(i);
+  }
+}
```
Comment (omar-muhamad) [inline script.js:52]: - [x] Same here nothing appears on the page so Please Make sure that when a user clicks the "Remove" button:
- The correct book is removed from the collection.
- The correct book disappears from the page.

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi Team,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [x] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by m05e5 — APPROVED** [review]
# Hi @gasangw  , 

### Status: Approved ✔️
goodjob


:+1: You have done all the required changes asked by the previous reviewer. Well Done.

Your project is complete! 
Congratulations! 🎉 


**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
-----

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_m05e5_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**